### PR TITLE
Update Mel Spectrogram Preprocessing

### PR DESCRIPTION
**Pull request by blisc**
Fix WER issue.

### CODE REVIEW

File: collections/nemo_asr/nemo_asr/audio_preprocessing.py [hunk 1]
```diff
@@ -274,6 +282,8 @@ def __init__(
             lowfreq=0,
             highfreq=None,
             log=True,
+            log_zero_guard_type="add",
+            log_zero_guard_value=2**-24,
```
Comment (blisc) [inline collections/nemo_asr/nemo_asr/audio_preprocessing.py]: @okuchaiev @vsl9 I'm not a fan of this default. I think we should switch to "tiny". But "tiny" results in a WER degradation from ~11.79% to ~12.00% WER on dev-other on the pretrained Jasper checkpoint.

Comment (okuchaiev) [reply collections/nemo_asr/nemo_asr/audio_preprocessing.py]: let's keep in mind that we want to move away from these defaults. Maybe open an issue to not forget? But for now, let's keep these defaults

Comment (ryanleary) [reply collections/nemo_asr/nemo_asr/audio_preprocessing.py]: Probably won't matter if you retrain. Hopefully in the future with NeMo we can version checkpoints and include metadata that specifies these kinds of things.

Comment (blisc) [reply collections/nemo_asr/nemo_asr/audio_preprocessing.py]: It is available as part of the yaml config file.
On a second thought, 'tiny' wouldn't be the best default as you can train a model in half precision but do inference in full precision or vice-versa resulting in different numbers.

**Pull request merged**